unit_tests/test_nemesis.py

```diff
@@ -223,3 +223,34 @@ def test_use_disabled_monkey():
     collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
 
     assert 'disrupt_toggle_table_gc_mode' in collected_disrupt_methods_names
+
+
+def test_list_all_available_nemesis(export=False):
+    import pickle
+
+    tester = FakeTester()
+
+    tester.params["nemesis_seed"] = '1'
+    tester.params["nemesis_selector"] = []
+    sisyphus = SisyphusMonkey(tester, None)
+
+    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
+
+    assert len(collected_disrupt_methods_names) == 67
```

Comment:
I wouldn't state the numbers in this test, just make this test fail if the generated and the static data doesn't match